app/src/main/java/org/oppia/android/app/home/HomeActivityPresenter.kt

```diff
@@ -50,4 +65,10 @@ class HomeActivityPresenter @Inject constructor(private val activity: AppCompatA
       R.id.home_fragment_placeholder
     ) as HomeFragment?
   }
+
+  private fun getSpotlightFragment(): SpotlightFragment? {
+    return activity.supportFragmentManager.findFragmentById(
+      R.id.home_spotlight_fragment_placeholder
+    ) as SpotlightFragment?
```

Comment:
```suggestion
    ) as? SpotlightFragment
```

Ditto everywhere else in the PR.

Reply:
done